css/src/components/table.scss

```diff
@@ -0,0 +1,68 @@
+$table-cell-vertical-padding: $spacer-4 !default;
+$table-cell-horizontal-padding: $spacer-5 !default;
+
+$table-caption-padding: $spacer-6 !default;
+$table-caption-size: $font-size-9 !default;
+$table-caption-spacing: $letter-spacing-medium !default;
+
+.table-wrapper {
+	overflow-x: auto;
+	overflow-y: hidden;
+	-webkit-overflow-scrolling: touch;
+}
+
+.table {
+	width: 100%;
+	font-size: $font-size-8;
+	border-collapse: collapse;
+	table-layout: auto;
+
+	caption {
+		padding: $table-caption-padding;
+		font-size: $table-caption-size;
+		font-weight: $weight-semibold;
+		letter-spacing: $table-caption-spacing;
+		text-align: $user-right;
+		text-transform: uppercase;
+	}
+
+	th,
+	td {
+		display: table-cell;
+		word-wrap: break-word;
+		padding: $table-cell-vertical-padding $table-cell-horizontal-padding;
+		border-top: 1px solid $border;
+		line-height: 1.5;
+		vertical-align: top;
+	}
+
+	thead th {
+		border-top: 0;
+		border-bottom: 1px solid $border;
+	}
+
+	th:not([align]),
+	td:not([align]) {
+		text-align: $user-left;
+	}
```

Comment:
I believe this is only needed for `th:not([align])` to ensure table headings are left-aligned by default rather than centered (the "none" case below).

Some test html and markdown:

| left | center | right | none |
|:-----|:------:|------:|------|
|😊    |😊     |😊    |😊    |
|aaaaaaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaa|

```html
<table>
<thead>
<tr>
<th align="left">left</th>
<th align="center">center</th>
<th align="right">right</th>
<th>none</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left">😊</td>
<td align="center">😊</td>
<td align="right">😊</td>
<td>😊</td>
</tr>
<tr>
<td align="left">aaaaaaaaaaaaaaaaaaaaa</td>
<td align="center">aaaaaaaaaaaaaaaaaaa</td>
<td align="right">aaaaaaaaaaaaaaaaa</td>
<td>aaaaaaaaaaaaaaaaa</td>
</tr>
</tbody>
</table>
```
```md
| left | center | right | none |
|:-----|:------:|------:|------|
|😊    |😊     |😊    |😊    |
|aaaaaaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaa|aaaaaaaaaaaaaaaaa|
```

Reply:
When I was testing it last week, I believe without these rules text was aligned in table cells to the right side, which was very surprising to me, but today I noticed as well that it takes effect on `th` only. Was it some testing accident? Maybe, I'm not sure :D 

I feel like for consistency I would prefer to keep it for both `td` and `th` but probably it's not needed.

Reply:
Updated